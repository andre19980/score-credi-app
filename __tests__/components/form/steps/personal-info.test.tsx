import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import PersonalInfoStep from "@/components/form/steps/personal-info";

describe("Personal Info Step", () => {
  it("renders form", () => {
    const { container } = render(<PersonalInfoStep />);
    expect(container).toMatchSnapshot();
  });
});
