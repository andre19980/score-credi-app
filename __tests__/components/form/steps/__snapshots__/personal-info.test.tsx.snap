// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Personal Info Step renders form 1`] = `
<div>
  <div>
    <h2
      class="text-base font-semibold leading-7 text-gray-900 mb-3"
    >
      Dados Pessoais
    </h2>
    <div
      class="grid grid-cols-1 gap-x-6 gap-y-4 sm:grid-cols-6 mt-4"
    >
      <div
        class="sm:col-span-4"
      >
        <label
          class="block text-sm font-medium leading-6 text-gray-900"
          for="name"
        >
          Nome
        </label>
        <div
          class="mt-2"
        >
          <div
            class="flex rounded-md shadow-sm ring-1 ring-inset ring-gray-300 focus-within:ring-2 focus-within:ring-inset focus-within:ring-indigo-600"
          >
            <input
              class="block flex-1 border-0 bg-transparent py-1.5 pl-1 text-gray-900 placeholder:text-gray-400 focus:ring-0 sm:text-sm sm:leading-6"
              id="name"
              type="text"
            />
          </div>
        </div>
      </div>
      <div
        class="sm:col-span-2"
      >
        <label
          class="block text-sm font-medium leading-6 text-gray-900"
          for="age"
        >
          Idade
        </label>
        <div
          class="mt-2"
        >
          <div
            class="flex w-full rounded-md shadow-sm ring-1 ring-inset ring-gray-300 focus-within:ring-2 focus-within:ring-inset focus-within:ring-indigo-600"
          >
            <input
              class="block flex-1 w-full border-0 bg-transparent py-1.5 pl-1 text-gray-900 placeholder:text-gray-400 focus:ring-0 sm:text-sm sm:leading-6"
              id="age"
              type="number"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
